<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ客户端</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="lv_history">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>191</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_sendcontent">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>300</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="lv_chatrecord">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>30</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chatName">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Hannotate SC&quot;;</string>
    </property>
    <property name="text">
     <string>聊天对象</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_notity">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>30</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Hannotate SC&quot;;</string>
    </property>
    <property name="text">
     <string>公告展示</string>
    </property>
   </widget>
   <widget class="QListView" name="lv_personnumber">
    <property name="geometry">
     <rect>
      <x>601</x>
      <y>140</y>
      <width>91</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lable_personnumber">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>116</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>成员0/0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>676</x>
      <y>119</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/new/prefix1/search_box_icon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_face">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>267</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>表情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_screanshot">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>267</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>截图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sendfile">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>267</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_voice">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>267</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>语音</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_video">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>267</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>视频</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_serachpn">
    <property name="geometry">
     <rect>
      <x>602</x>
      <y>118</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lw_singlerescord">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>29</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_singalsendcontent">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>299</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>491</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/client_bg.jpg);
font: 75 20pt &quot;Hannotate SC&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>欢迎使用聊天系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addroom">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>addRoom</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_welcome</zorder>
   <zorder>le_serachpn</zorder>
   <zorder>btn_sendfile</zorder>
   <zorder>btn_search</zorder>
   <zorder>btn_face</zorder>
   <zorder>te_sendcontent</zorder>
   <zorder>label_notity</zorder>
   <zorder>btn_screanshot</zorder>
   <zorder>lv_personnumber</zorder>
   <zorder>btn_video</zorder>
   <zorder>lable_personnumber</zorder>
   <zorder>btn_voice</zorder>
   <zorder>lv_chatrecord</zorder>
   <zorder>lv_history</zorder>
   <zorder>chatName</zorder>
   <zorder>lw_singlerescord</zorder>
   <zorder>te_singalsendcontent</zorder>
   <zorder>btn_addroom</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
